Add Studio Link Plugin v15.12.1-beta

lukasbestle opened the pull request.

vitorgalvao commented on Casks/studio-link-plugin.rb:
```diff
@@ -0,0 +1,13 @@
+  version '15.12.1-beta'
+  sha256 '12145c5d054527a1460038166f5cf2d18219e3b614e01391af9b8b6d7a3d765f'
+
+  url "https://github.com/Studio-Link-v2/backend/releases/download/#{version}/studio-link-osx.zip"
```
Please add a `appcast 'https://github.com/Studio-Link-v2/backend/releases.atom'` line under.

vitorgalvao replied:
In addition, please point to where on the website you found the download link.

vitorgalvao replied:
Needs a `# github.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/7c010b42bd7ac679c1f002a0368b36bb3f495612/doc/CASK_LANGUAGE_REFERENCE.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

lukasbestle commented:
Thanks for the feedback. I just added the missing lines and squashed my commits.

Since the plugin is currently in beta, the documentation is not already on the main site but on a [subdomain](https://doku.studio-link.de/plugin/installation-plugin.html).

vitorgalvao commented:
> Since the plugin is currently in beta, the documentation and download link is not already on the main site but on a [subdomain](https://doku.studio-link.de/plugin/installation-plugin.html).

Ah, we should use that for now, then.

vitorgalvao commented:
Thank you for the contribution. It needed some fixes, so they were made in b6e6c9589b3c287d7b76ec76bd263baf1a4b7f64. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

lukasbestle commented:
Thanks! Looks good.
